frontend/src/container/FormAlertRules/BasicInfo.tsx

```diff
@@ -105,18 +136,38 @@ function BasicInfo({ alertDef, setAlertDef }: BasicInfoProps): JSX.Element {
 						initialValues={alertDef.labels}
 					/>
 				</FormItemMedium>
-				<FormItemMedium label="Notification Channels">
-					<ChannelSelect
-						currentValue={alertDef.preferredChannels}
-						onSelectChannels={(preferredChannels): void => {
-							setAlertDef({
-								...alertDef,
-								preferredChannels,
-							});
-						}}
+
+				<FormItemMedium
+					name="alert_all_configured_channels"
+					label="Alert all the configured channels"
+				>
+					<Switch
+						checked={shouldBroadCastToAllChannels}
+						onChange={handleBroadcastToAllChannels}
 					/>
-					<ChannelSelectTip> {t('channel_select_tooltip')}</ChannelSelectTip>
 				</FormItemMedium>
+
+				{!shouldBroadCastToAllChannels && (
```

Comment:
shall we disable the input element rather than removing it from UI? It will remove the flashy feel from the form

Reply:
We will have to clear the previously selected channels and if the user toggles again, the user has to reselect from the start.